apps/SageAccountWeb/src/components/AccountSettings.tsx

```diff
@@ -399,24 +399,17 @@ export const AccountSettings = () => {
                       })}
                     </Grid>
                   </StyledFormControl>
-                  <StyledFormControl
+                  <SRCTextField
                     fullWidth
-                    variant="standard"
                     margin="normal"
-                    sx={{ marginBottom: '10px' }}
-                  >
-                    <InputLabel shrink htmlFor="bio">
-                      Bio
-                    </InputLabel>
-                    <TextField
-                      id="bio"
-                      name="bio"
-                      fullWidth
-                      multiline
```

Comment:
The multiline styling broke here, but this is the only place where it's used, so just use the SRCTextField, which handles it properly.